server/migrations/1629329748833_feed-database.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable camelcase */
+const { readFileSync } = require('fs');
+const { join } = require('path');
+const readXlsxFile = require('node-xlsx').default;
+
+const { dbClient, collections } = require('../db');
+const { createRows, verifyHeaders } = require('../utils');
+
+const siteColumnMap = {
+  'HCAP Site ID': 'siteId',
+  'Site Name': 'siteName',
+  RHO: 'isRHO',
+  Allocation: 'allocation',
+  'Street Address': 'address',
+  'Health Authority': 'healthAuthority',
+  City: 'city',
+  'Post Code': 'postalCode',
+  'Registered Business Name': 'registeredBusinessName',
+  'Operator Name': 'operatorName',
+  'Operator Contact First Name': 'operatorContactFirstName',
+  'Operator Contact Last Name': 'operatorContactLastName',
+  'Operator Contact Email': 'operatorEmail',
+  'Operator Contact Phone': 'operatorPhone',
+  'Site Contact First Name': 'siteContactFirstName',
+  'Site Contact Last Name': 'siteContactLastName',
+  'Site Contact Phone Number': 'siteContactPhoneNumber',
+  'Site Contact Email': 'siteContactEmailAddress',
+};
+
+const participantColumnMap = {
+  ClientID: 'maximusId',
+  Surname: 'lastName',
+  Name: 'firstName',
+  PostalCode: 'postalCode',
+  'Post Code FSA': 'postalCodeFsa',
+  Phone: 'phoneNumber',
+  Email: 'emailAddress',
+  'EOI - FHA': 'fraser',
+  'EOI - IHA': 'interior',
+  'EOI - NHA': 'northern',
+  'EOI - VCHA': 'vancouverCoastal',
+  'EOI - VIHA': 'vancouverIsland',
+  'CB1: Still Interested': 'interested',
+  'CB8: Non-HCAP Opportunities': 'nonHCAP',
+  'CB13: CRC Clear': 'crcClear',
+};
+
+exports.up = async () => {
+  const nodeEnv = process.env.NODE_ENV;
+  if (nodeEnv === 'test' || nodeEnv === 'local' || nodeEnv === 'dev') {
```

Comment:
We should be careful here.  This will seed for local and in tests just fine, but for all deployed environments NODE_ENV will be `production`. If we plan on doing any seeding on `dev`/`test`, we should wait for HCAP-870 to be complete so we have an accurate environment specific variable.

Reply:
Fully willing to shelve this to wait for HCAP-870. In hindsight, however, it's only really needed for the 'local' and 'test' environments, the latter being what gets run within the pipeline.